app/bills/page.tsx

```diff
@@ -48,74 +47,101 @@ export default async function BillSearchPage({
     searchParams.state
   );
 
+  console.log("bills", bills);
+
   const query = searchParams.query;
-  const state = searchParams.year;
-  const year = searchParams.state;
+  const state = searchParams.state;
+  const year = searchParams.year;
 
   const params = new URLSearchParams();
 
   if (query) params.append("query", query);
-  if (state) params.append("state", state);
   if (year) params.append("year", year);
+  if (state) {
+    let states: string[] = [];
+
+    if (Array.isArray(state)) {
```

Comment:
When is the state query an array vs a string delineated by commas? @JonathanTylerWeber